plugin/externalPluginScript.go

```diff
@@ -62,9 +62,9 @@ func (s externalPluginScriptImpl) InvokeAndWait(
 		return inv, newInvokeError(err.Error(), inv)
 	}
 
-	activity.Record(ctx, "stdout: %v", inv.stdout)
+	activity.Record(ctx, "stdout: %v", inv.stdout.String())
```

Comment:
Wondering if we should reduce the verbosity here. When I do something like
```
➜  mnt107015990 stree
INFO Invoking /Users/enis.inan/GitHub/wash/schema.rb schema /schema ''
INFO stdout: {
  "root": {
  	"label": "schema",
  	"methods": ["list","schema"],
  	"singleton": true,
  	"children": [
  		"A"
  	]
  },
  "A": {
  	"label": "NodeA",
  	"methods": ["list","schema"],
  	"singleton": true,
  	"children": [
  		"B"
  	]
  },
  "B": {
  	"label": "NodeB",
  	"methods": ["read","schema"]
  }
}
.
├── docker
│   ├── containers
│   │   └── [container]
│   │       ├── log
│   │       ├── metadata.json
│   │       └── [fs]
│   │           ├── [dir]
│   │           │   ├── [dir]
│   │           │   └── [file]
│   │           └── [file]
│   └── volumes
│       └── [volume]
│           ├── [dir]
│           │   ├── [dir]
│           │   └── [file]
│           └── [file]
└── schema
    └── NodeA
        └── [NodeB]
```

It's a bit annoying having to see stdout all the time.

Reply:
It seems wrong that we’re trying to write to activity on the CLI side.

Reply:
Yeah, that's a good point -- that output would add-up the more external plugins you have. Should we file an issue about it?

Reply:
I'm confused why this code is being exercised client-side in the first place.

Reply:
I think it might have been during the time the activity wrote to the debug logs for all journals, so I put it there to make debugging easier.

Reply:
Also, rather than calling `.String()` we could replace `%v` with `%s` (which should do the same thing).

Reply:
Looks like it doesn't. Here's what I get when I use `%s`:
```
INFO stdout: {{
  "type_id": "root",
	"name": "schema",
	"methods": [
			"schema",
		"list"
	]
}
 %!s(int=0) %!s(bytes.readOp=0)}
```

vs `%v` + `String()`:
```
INFO stdout: {
  "type_id": "root",
	"name": "schema",
	"methods": [
			"schema",
		"list"
	]
}
```

Reply:
Huh, that's surprising.